br/pkg/lightning/restore/restore.go

```diff
@@ -975,6 +975,11 @@ func (rc *Controller) saveStatusCheckpoint(ctx context.Context, tableName string
 	case !common.IsContextCanceledError(err):
 		merger.SetInvalid()
 		rc.errorSummaries.record(tableName, err, statusIfSucceed)
+	case utils.IsRetryableError(err), utils.MessageIsRetryableStorageError(err.Error()):
+		// recoverable error, should not be recorded in checkpoint
+		// which will prevent lightning from automatically recovering
+		rc.errorSummaries.record(tableName, err, statusIfSucceed)
```

Comment:
should we record this error? since it's recoverable

errorSummaries is emitted after `controller.Run`.